src/google/cloud/ndb/model.py

```diff
@@ -3362,29 +3373,29 @@ class StructuredProperty(Property):
     The values of the sub-entity are indexed and can be queried.
     """
 
-    _modelclass = None
+    _model_class = None
```

Comment:
In different places, it was spelled ``_modelclass``, ``_model_class``, and ``_kls``. I picked one.